### ISSUE TITLE
All Components - Some text is too small

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9964]
As a user with low vision, I expect all text in the page to be of an appropriate size to ensure easy reading. 

**Linked work item:** 
#9391 #9387  

**Expected Result:**
All text should be an appropriate size to ensure users with low vision can easily read text.

**Actual Result:**
Text at the bottom of the left nav and 'Details'/'Related Resources' text on each search result is too small. Text which is very small is difficult to read, particularly for those with low vision. 

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/d2866242-3b35-455e-9e4c-945b271b6a26)

**Comment by johnatawnclementawn** [comment]
Resolved in #10078
